curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-090.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-90
 
 Now that you're familiar with the `box-shadow` property you can finalize the shadows, starting with the one for the red marker.
 
-In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, and `spreadRadius` is `0`.
+In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, `spreadRadius` is `0` and `color` is `red`.
```

Comment:
```suggestion
In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, `spreadRadius` is `0`, and `color` is `red`.
```
Nitpicky, but I'm a huge proponent of the Oxford comma. 🙂

Reply:
np, will commit that change.